101-databricks-workspace/azuredeploy.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "workspaceName": {
+          "value": "GEN-UNIQUE"
+        },
+        "pricingTier": {
```

Comment:
remove this and just use the defaultValue in the template

Reply:
Done.